applications/fishing-map/src/features/map/controls/MapSearch.tsx

```diff
@@ -2,36 +2,24 @@ import React, { memo, useRef, useState } from 'react'
 import cx from 'classnames'
 import { useTranslation } from 'react-i18next'
 import { useCombobox, UseComboboxStateChange } from 'downshift'
-import { fitBounds } from 'viewport-mercator-project'
 import InputText from '@globalfishingwatch/ui-components/dist/input-text'
 import IconButton from '@globalfishingwatch/ui-components/dist/icon-button'
 import { OceanArea, searchOceanAreas } from '@globalfishingwatch/ocean-areas'
-import { useMapboxInstance } from '../map.context'
-import useViewport from '../map-viewport.hooks'
+import { useMapFitBounds } from '../map-viewport.hooks'
 import styles from './MapSearch.module.css'
 
 const MapSearch = () => {
   const { t } = useTranslation()
   const [query, setQuery] = useState<string>('')
   const inputRef = useRef<HTMLInputElement | null>(null)
   const [areasMatching, setAreasMatching] = useState<OceanArea[]>([])
-  const mapInstance = useMapboxInstance()
 
-  const { setMapCoordinates } = useViewport()
+  const fitBounds = useMapFitBounds()
 
   const onSelectResult = ({ selectedItem }: UseComboboxStateChange<OceanArea>) => {
-    const bounds = selectedItem?.properties.bounds
+    const bounds = selectedItem?.properties.bounds as [number, number, number, number]
```

Comment:
Minor, I checked and there are already 8 occurrences of `[number, number, number, number]` in `fishing-map`. I suggest we use 
```
import { BBox } from 'geojson'
``` 
What do you think?

Reply:
I had to do it because the type definition of bbox in geojson package is
```
export type BBox = [number, number, number, number] | [number, number, number, number, number, number];
```

that is way I needed to define which of both formats we are using, maybe we can just create our own definition of BBox as store in `/types`  WDYT ?

Reply:
Perfect

Reply:
👍 https://github.com/GlobalFishingWatch/frontend/commit/7ebe184dfc4513b7cb1f50503202ad3c23103997